src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/RespondToDefenceCallbackHandlerTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.unspec.handler.callback;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.callback.CallbackType;
+import uk.gov.hmcts.reform.unspec.enums.YesOrNo;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {
+    RespondToDefenceCallbackHandler.class,
+    JacksonAutoConfiguration.class
+})
+class RespondToDefenceCallbackHandlerTest extends BaseCallbackHandlerTest {
+
+    @Autowired
+    private RespondToDefenceCallbackHandler handler;
+
+    @Nested
+    class SubmittedCallback {
+        public static final String APPLICANT_1_PROCEEDING = "applicant1ProceedWithClaim";
```

Comment:
can we use this from `RespondToDefenceCallbackHandler` itself?

Reply:
no, I makes the test more meaningful if this is also defined here imo